test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -672,132 +669,6 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
         }
 
         [Fact]
-        public async Task CanReceiveDataEvenIfExceptionThrownFromPreviousReceivedEvent()
```

Comment:
I think these tests are still relevant - user code can throw and I don't see exceptions to be handled.